sqlparse/filters.py

```diff
@@ -690,9 +690,14 @@ def _create_column_definition(self, token_list):
         column_definition_children_tokens.extend(
             self._skip_white_space_and_new_line_tokens(token_queue)
         )
-        column_type_length_token = self._create_column_type_length(token_queue)
-        if column_type_length_token:
-            column_definition_children_tokens.append(column_type_length_token)
+
+        if (col_type_token.value.lower() == 'set' or
+                col_type_token.value.lower() == 'enum'):
+            column_type_meta_token = self._create_column_type_values(token_queue)
+        else:
+            column_type_meta_token = self._create_column_type_length(token_queue)
+        if column_type_meta_token:
```

Comment:
Is this `if` required here? Won't `column_type_meta_token` be set in the above `if` `else`?